rdbms-image/devicehive-backend/Dockerfile

```diff
@@ -18,7 +23,7 @@ RUN cd /usr/share \
 ENV MAVEN_HOME /usr/share/maven
 
 # download devicehive and build .jar
-RUN git clone -b development https://github.com/devicehive/devicehive-java-server.git /opt/devicehive && \
+RUN git clone -b master https://github.com/devicehive/devicehive-java-server.git /opt/devicehive && \
```

Comment:
We should not clone repo and build the artifact inside Dockerfile. It's better to use prebuilt jar file. Because this cause a lot of unnecessary garbage inside container and it became too heavy.